### PR TITLE
Initialize Routing when not done before Exporting. 

### PR DESCRIPTION
**Pull request by mchf**
bnc#956012

### CODE REVIEW

File: src/modules/Routing.rb [hunk 1]
```diff
@@ -391,6 +394,12 @@ def Import(settings)
     def Export
       exproute = {}
 
+      # It should be case only for installer (1st stage). When routing
+      # was configured via linuxrc, yast needn't to be aware of it. bnc#956012
+      Routing.Read if !@initialized
```
Comment (jreidinger) [inline src/modules/Routing.rb]: simple `Read() if !@initialized` is enough as it is same module

**Comment by jreidinger** [comment]
just NP, LGTM

File: package/yast2-network.changes [hunk 2]
```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Mon Nov 23 09:54:34 UTC 2015 - mfilka@suse.com
+
+- bnc#956012
+  - routing configuration exported at the end of first stage even
+    when configured via linuxrc
```
Comment (mvidner) [inline package/yast2-network.changes]: It is unclear whether this describes the bug or the fix :) Also, can you please mention the user visible symptoms?

**Comment by kobliha** [comment]
@mchf @schubi2 PLS add this use-case to AutoYast autotest

**Comment by mvidner** [comment]
LGTM

**Comment by mchf** [comment]
Thanks for reviews

**Pull request merged**